File: org.amanzi.splash/src/org/amanzi/splash/jruby/Spreadsheet.rb
class Cells
  def initialize
    @cells = {'a1' => 'Hello'};     
  end
  
    def method_missing(meth, *args)
      s = "#{meth}"
      s = s.gsub("=","")
      if (args.to_s.eql?"")
        @cells[s]
      else
        @cells[s] = args.to_s
      end
    end
end

class Spreadsheet
  def initialize
    @cells = Cells.new
  end
  
  def cells
    @cells
  end
end

File: org.amanzi.splash/src/org/amanzi/splash/jruby/Spreadsheet.java
package org.amanzi.splash.jruby;

import org.amanzi.splash.utilities.Util;

public class Spreadsheet {
	public JRubyJavaInterface cells;
	public String test;
	
	public Spreadsheet() {
		super();
		cells = new JRubyJavaInterface();
		test = "Hello";
	}
	
	public static void print_somthing()
	{
		Util.log("Hello, I'm java");
	}

	public JRubyJavaInterface cells()
	{
		return cells;
	}
}
